CONTRIBUTING.md

```diff
@@ -304,6 +303,31 @@ same name.
 </record>
 ```
 
+
+Use of `<... position="replace">` is not recommended because
+could show the error `Element ... cannot be located in parent view`
+from other views inheriting with this field.
+
+if you need to use this option, must have an explicit comment
+explaining why they are absolutely necessary and use a
+high value of priority to avoid the error.
```

Comment:
[Why >100 is okay](https://github.com/OCA/maintainer-tools/pull/206/files#diff-6a3371457528722a734f3c51d9238c13R320)?, in any case, I think is would be a good idea to put in it.
> " high value in its `priority` (greater than 100 is recommended) to avoid the error."